src/main/java/com/amazon/exception/ExceptionControllerAdvice.java

```diff
@@ -0,0 +1,19 @@
+package com.amazon.exception;
+
+import com.amazon.dto.UserDto;
+import com.amazon.dto.WebApiResult;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionControllerAdvice {
+
+    @ExceptionHandler
+    public ResponseEntity<WebApiResult<UserDto>> invalidUserExceptionHandler(RuntimeException exception){
```

Comment:
Create a user defined exception "InvalidUserException" and throw that when user is invalid